src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -451,10 +451,17 @@ private void visitEnumType(EnumType type) {
       emitBreak();
     }
 
-    private void visitTypeDeclaration(JSType type) {
+    private void visitTypeDeclaration(JSType type, boolean isVarArgs) {
       if (type != null) {
         emit(":");
-        visitType(type);
+        if (isVarArgs && !type.isAllType() && !type.isUnknownType()) {
```

Comment:
True, the downside is that there is implicit assumption that type.isAllType and type.isUnknownType's output is safe to emit without parens, which is certainly true now, but might change in the future.

I don't feel strongly here, so keep as it is.